tests/ert3/console/integration/test_cli.py

```diff
@@ -336,10 +264,42 @@ def test_cli_export_uniform_polynomial_evaluation(tmpdir):
     with unittest.mock.patch.object(sys, "argv", args):
         ert3.console.main()
 
-    _assert_uniform_export(workspace, "uniform_evaluation")
+    _assert_export(workspace, "uniform_evaluation")
 
 
-def test_cli_run_presampled(tmpdir):
+def _assert_distribution(workspace, experiment, distribution, coefficients):
+    samples = {idx: [] for idx in ("a", "b", "c")}
```

Comment:
The assumption of coefficients being named `a, b, c` is repeated twice. The assumption is fine, but perhaps we could store it in a variable? ie:

```python
coefficients_names = ("a", "b", "c")
```